examples/gnrc_networking/Makefile

```diff
@@ -13,11 +13,8 @@ USEMODULE += netdev_default
 USEMODULE += auto_init_gnrc_netif
 # Activate ICMPv6 error messages
 USEMODULE += gnrc_icmpv6_error
-# Specify the mandatory networking module for a IPv6 routing node
-USEMODULE += gnrc_ipv6_router_default
-# Add a routing protocol
-USEMODULE += gnrc_rpl
-USEMODULE += auto_init_gnrc_rpl
+# Specify the mandatory networking module for a IPv6 host node
+USEMODULE += gnrc_ipv6_default
```

Comment:
Why remove the router “feature” altogether? I would prefer a

```Makefile
ROUTER ?= 0
```

switch

Reply:
Sounds like a good idea. 👍